src/components/AttachmentModal.js

```diff
@@ -42,28 +45,91 @@ const propTypes = {
 };
 
 const defaultProps = {
-    title: '',
-    sourceURL: null,
-    onConfirm: null,
-    isAuthTokenRequired: false,
+    sortedReportActions: [],
+    visibleAttachment: {
+        sourceURL: null,
+        isAttachment: false,
+        isModalOpen: false,
+        file: {name: ''},
+    },
 };
 
 class AttachmentModal extends Component {
     constructor(props) {
         super(props);
 
+        this.handleKeyPress = this.handleKeyPress.bind(this);
+        this.getNextAttachment = this.getNextAttachment.bind(this);
+        this.getAttachmentSource = this.getAttachmentSource.bind(this);
+
         this.state = {
-            isModalOpen: false,
-            file: null,
-            sourceURL: props.sourceURL,
+            title: 'Attachment',
         };
     }
 
+    componentDidUpdate(prevProps) {
+        const {visibleAttachment: {isModalOpen}} = this.props;
+        if (!prevProps.visibleAttachment.isModalOpen && isModalOpen) {
+            document.addEventListener('keydown', this.handleKeyPress, false);
+        } else {
+            document.removeEventListener('keydown', this.handleKeyPress, false);
```

Comment:
I think `document` does not exist in native environments. So if you want to use it we'd need to have 2 versions of this component. `KeyboardShortcut` will solve this problem for you.

Reply:
KeyboardShortcuts unsubscribe wouldn't work for some reason, which was why I was forced to use document

Reply:
To clarify, `document` _cannot_ be used. It is breaking native platforms...

![2021-01-13_11-02-08](https://user-images.githubusercontent.com/32969087/104510137-ca23e880-558e-11eb-809c-e4d726c5b6f2.png)